ops/swagger_to_openapi.sh

```diff
@@ -8,4 +8,8 @@ set -o pipefail
 java -jar -DswaggerUrl=swagger.yaml /swagger-converter/jetty-runner.jar /swagger-converter/server.war &
 sleep 30
 curl -X POST -d "@swaggerui/swagger.json" localhost:8080/api/convert -H "Content-Type: application/json" -o temp.json
-cat temp.json | jq > swaggerui/openapi.json
\ No newline at end of file
+cat temp.json | jq > swaggerui/openapi.json
+
+# Remove openapi.json file if it is invalid
+# Note that build_and_package.sh script validates that openapi.json exists; so RPM packaging will fail if there is an invalid openapi.json
+grep "Beneficiary Claims Data API" swaggerui/openapi.json || rm swaggerui/openapi.json
```

Comment:
If we need the openapi.json file to exist for the RPM stage to succeed, should we fail this script instead of removing the file? That way, we would catch the failure earlier in the process (here vs. at the RPM stage)